src/main/java/com/rybka/config/MapSearchUtil.java

```diff
@@ -0,0 +1,12 @@
+package com.rybka.config;
+
+import java.util.Map;
+
+public class MapSearchUtil {
+    public static <T> Map.Entry<String, T> retrieveMapValue(Map<String, T> map, String searchItem, RuntimeException runtimeException) {
+        return map.entrySet().stream()
```

Comment:
Again, use map.get() instead of streams.

Reply:
fixed